src/test/java/in/balendin/schoolmanagment/testcases/organization/OrganizationTestCase.java

```diff
@@ -0,0 +1,90 @@
+package in.balendin.schoolmanagment.testcases.organization;
+
+import in.balendin.schoolmanagment.steps.LoginSteps;
+import in.balendin.schoolmanagment.steps.OrganizationSteps;
+import in.balendin.schoolmanagment.testcases.basetest.BaseTestCase;
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Narrative;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.Title;
+import net.thucydides.core.annotations.WithTagValuesOf;
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+
+@Narrative(text = {"As a super admin",
+"I want to be able to redirect organizationList when super admin clicks on GoBack button"})
+@RunWith(SerenityRunner.class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+@WithTagValuesOf({"organization","regression"})
+public class OrganizationTestCase extends BaseTestCase {
+
+    @Steps
+    LoginSteps loginSteps;
+
+    @Steps
+    OrganizationSteps organizationSteps;
+
+    @Before
+    public void loginAsSuperAdmin(){
+        loginSteps.navigateToApplication();
+        loginSteps.loginAsSuperAdmin();
+    }
+    @Test
+    @Title("Verify super admin can create organization with mandatory fields")
+    public void superAdminIsCreateOrganization(){
+        organizationSteps.createOrganization();
+    }
+    @Test
+    @Title("Verify super admin can able to see organization list ")
+    public void superAdminIsSeeOrganizationList(){
+        organizationSteps.superAdminIsRedirectSuccessfulOnOrganizationList();
+    }
+    @Test
+    @Title("Verify super admin can able to search organization name")
```

Comment:
@pratik-9839 as we discussed please merge creating and deleting testcases as part of this

Reply:
@rajaYendluri completed as per your discussion